Index: vendor/lara-zeus/sky/src/Models/Post.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vendor/lara-zeus/sky/src/Models/Post.php b/vendor/lara-zeus/sky/src/Models/Post.php
--- a/vendor/lara-zeus/sky/src/Models/Post.php
+++ b/vendor/lara-zeus/sky/src/Models/Post.php	(date 1755647650652)
@@ -70,7 +70,7 @@
         return [
             'published_at' => 'datetime',
             'sticky_until' => 'datetime',
-            'status' => config('zeus-sky.models.PostStatus'),
+            'status' => config('zeus-sky.enums.PostStatus'),
         ];
     }
 
