Index: vendor/lara-zeus/sky/src/Filament/Resources/PostResource.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vendor/lara-zeus/sky/src/Filament/Resources/PostResource.php b/vendor/lara-zeus/sky/src/Filament/Resources/PostResource.php
--- a/vendor/lara-zeus/sky/src/Filament/Resources/PostResource.php
+++ b/vendor/lara-zeus/sky/src/Filament/Resources/PostResource.php	(date 1755645500509)
@@ -118,7 +118,7 @@
                                     ->default('publish')
                                     ->required()
                                     ->live()
-                                    ->options(SkyPlugin::get()->getModel('PostStatus')),
+                                    ->options(SkyPlugin::get()->getEnum('PostStatus')),
 
                                 TextInput::make('password')
                                     ->label(__('Password'))
